src/domains/citizen-auth-common/01_network.tf

```diff
@@ -46,14 +46,34 @@ data "azurerm_private_dns_zone" "privatelink_table_core" {
   resource_group_name = local.vnet_common_resource_group_name
 }
 
+data "azurerm_subnet" "private_endpoints_subnet_itn" {
+  name                 = "io-p-itn-pep-snet-01"
+  virtual_network_name = "io-p-itn-common-vnet-01"
+  resource_group_name  = "io-p-itn-common-rg-01"
+}
 
 ## Redis Common subnet
 module "redis_common_snet" {
-  source               = "git::https://github.com/pagopa/terraform-azurerm-v3.git//subnet?ref=v7.62.0"
+  source               = "git::https://github.com/pagopa/terraform-azurerm-v3.git//subnet?ref=v8.44.0"
   name                 = format("%s-redis-snet", local.project)
   address_prefixes     = var.cidr_subnet_redis_common
   resource_group_name  = local.vnet_common_resource_group_name
   virtual_network_name = local.vnet_common_name
 
   private_endpoint_network_policies_enabled = false
 }
+
+## Cosmos Private Endpoint
+resource "azurerm_private_endpoint" "cosmos_db" {
+  name                = "${local.product}-itn-citizen-auth-account"
+  location            = "italynorth"
+  resource_group_name = azurerm_resource_group.data_rg.name
+  subnet_id           = data.azurerm_subnet.private_endpoints_subnet_itn.id
+
+  private_service_connection {
+    name                           = "${local.product}-itn-citizen-auth-account-private-endpoint"
```

Comment:
Ditto, same value as above

Reply:
fixed in [7c6c596](https://github.com/pagopa/io-infra/pull/1188/commits/7c6c5968af2b018054b3d6cb89f3cc73ad568177)